packages/auth/app/templates/spree/account.hbs

```diff
@@ -10,3 +10,32 @@
     </form>
   </div>
 </div>
+<div class="row">
+  <h3>My Orders</h3>
+  <table class="small-12 columns">
+    <thead class="active">
+      <tr>
+        <th class="order-number">Number</th>
+        <th class="order-date">Date</th>
+        <th class="order-status">Status</th>
+        <th class="order-payment-state">Payment State</th>
+        <th class="order-shipment-state">Shipment State</th>
+        <th class="order-total">Total</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each order in session.currentUser.orders}}
```

Comment:
can we change this to `{{#each session.currentUser.orders as |order|}}`?

apparently this syntax means Ember.JS apps running Glimmer will render faster